### ISSUE TITLE
Clarification on TAP Task lifecycle

### ISSUE THREAD

**Issue by ling1726** [dotnet/docs#15987]
*All other tasks begin their life cycle in a hot state, which means that the asynchronous operations they represent have already been initiated and their task status is an enumeration value other than TaskStatus.Created. All tasks that are returned from TAP methods must be activated. **If a TAP method internally uses a task’s constructor to instantiate the task to be returned, the TAP method must call Start on the Task object before returning it.** Consumers of a TAP method may safely assume that the returned task is active and should not try to call Start on any Task that is returned from a TAP method. Calling Start on an active task results in an InvalidOperationException exception.*

The above extract of the documentation seems to not be 100% clear. I would like to get clarification that the following points:

* an `active` Task in this section is equivalent to `hot` Task. 
* Activating a Task would imply using calling `Start` on a Task created by a constructor

For context, we are currently using a `Fire and forget` async method and would like some kind of confirmation that the Task is guaranteed to be started.

Thanks,

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e40c0699-0e7c-e20d-98c1-76f0520ab9c2
* Version Independent ID: 0fc31902-0055-f9e8-a228-e7a0ce384e35
* Content: [Task-based Asynchronous Pattern (TAP)](https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap?redirectedfrom=MSDN#feedback)
* Content Source: [docs/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap.md](https://github.com/dotnet/docs/blob/master/docs/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap.md)
* Product: **dotnet**
* Technology: **dotnet-standard**
* GitHub Login: @BillWagner
* Microsoft Alias: **dotnetcontent**

**Comment by BillWagner** [comment]
> an active Task in this section is equivalent to hot Task.

Yes, we should standardize on "active Task" to match the enum names.

> Activating a Task would imply using calling Start on a Task created by a constructor.

Yes, but that's not how I usually explain it. Most often, you'd call some method that starts a task (either some `*Async` method, or `Task.Run`).  Those tasks are active when returned. The primary reason to call `new Task(...)` is to create a task that hasn't started , enabling you to start it later.
